cephbackupstorage/cephbackupstorage/cephagent.py

```
@@ -1002,19 +1002,17 @@ def _getRealSize(length):
-                return length
+                return int(length)
             units = {
                 "g": lambda x: x * 1024 * 1024 * 1024,
                 "m": lambda x: x * 1024 * 1024,
                 "k": lambda x: x * 1024,
             }
             try:
-                if not length[-1].isalpha():
-                    return length
                 return units[length[-1].lower()](int(length[:-1]))
             except:
```

Ruff (0.15.6)

[error] Do not use bare `except`

(E722)